experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -0,0 +1,540 @@
+        error("land temp anomaly function $land_temperature_anomaly not supported")
+    temp_anomaly = T_functions[land_temperature_anomaly]
+    T_sfc0 = FT(276.85) .+ temp_anomaly.(coords.subsurface)
+    lapse_rate = FT(6.5e-3)
+    # Adjust initial temperature to account for orography of the surface
+    # `surface_elevation` is a ClimaCore.Fields.Field(`half` level)
+    orog_adjusted_T_data = CC.Fields.field_values(T_sfc0) .- lapse_rate .* CC.Fields.field_values(surface_elevation)
+    orog_adjusted_T = CC.Fields.Field(orog_adjusted_T_data, subsurface_space)
+    orog_adjusted_T_surface = CC.Fields.Field(CC.Fields.level(orog_adjusted_T_data, 1), surface_space)
+
+    @. Y.soil.ϑ_l = θ_r + (ν - θ_r) / 2
+    Y.soil.θ_i .= FT(0.0)
+    ρc_s = CL.Soil.volumetric_heat_capacity.(Y.soil.ϑ_l, Y.soil.θ_i, soil_args.parameters.ρc_ds, earth_param_set)
+    Y.soil.ρe_int .= CL.Soil.volumetric_internal_energy.(Y.soil.θ_i, ρc_s, orog_adjusted_T, earth_param_set)
+    Y.soilco2.C .= FT(0.000412)
+
+    plant_ν = FT(1.44e-4)
+    Y.canopy.hydraulics.ϑ_l.:1 .= plant_ν
+    @. Y.canopy.energy.T = orog_adjusted_T_surface
+    Y.snow.S .= 0.0
+    Y.snow.S_l .= 0.0
+    Y.snow.U .= 0.0
+
+    set_initial_cache! = CL.make_set_initial_cache(model)
+    exp_tendency! = CL.make_exp_tendency(model)
+    imp_tendency! = CL.make_imp_tendency(model)
+    jacobian! = CL.make_jacobian(model)
+    set_initial_cache!(p, Y, tspan[1])
+
+    # set up jacobian info
+    jac_kwargs = (; jac_prototype = CL.FieldMatrixWithSolver(Y), Wfact = jacobian!)
+
+    prob = SciMLBase.ODEProblem(
+        CTS.ClimaODEFunction(
+            T_exp! = exp_tendency!,
+            T_imp! = SciMLBase.ODEFunction(imp_tendency!; jac_kwargs...),
+            dss! = CL.dss!,
+        ),
+        Y,
+        tspan,
+        p,
+    )
+
+    # Set up diagnostics
+    if use_land_diagnostics
+        netcdf_writer = CD.Writers.NetCDFWriter(subsurface_space, output_dir; start_date)
+        scheduled_diagnostics = CL.default_diagnostics(
+            model,
+            start_date,
+            output_writer = netcdf_writer,
+            output_vars = :short,
+            average_period = :monthly,
+        )
+        diagnostic_handler = CD.DiagnosticsHandler(scheduled_diagnostics, Y, p, tspan[1]; dt = dt)
+        diag_cb = CD.DiagnosticsCallback(diagnostic_handler)
+    else
+        diag_cb = nothing
+    end
+
+    # Set up time stepper and integrator
+    ode_algo =
+        CTS.IMEXAlgorithm(stepper, CTS.NewtonsMethod(max_iters = 3, update_j = CTS.UpdateEvery(CTS.NewNewtonIteration)))
+    integrator = SciMLBase.init(
+        prob,
+        ode_algo;
+        dt = dt,
+        saveat = saveat,
+        adaptive = false,
+        callback = SciMLBase.CallbackSet(diag_cb),
+    )
+
+    sim = ClimaLandSimulation(model, (; domain = domain, soil_depth = depth), integrator, area_fraction)
+
+    # DSS state to ensure we have continuous fields
+    dss_state!(sim)
```

Comment:
the land model does not use dss in standalone runs - are we sure we need this?

Reply:
A couple years ago we had conservation issues because some fields were discontinuous. We maintain continuous fields throughout the simulation, but we also need to ensure that the initial conditions are continuous. To make sure this is the case, we dss everything at the start of the simulation. It's possible that we don't need this for the land model with these ICs, but we do this across all component models so we don't miss anything.

This issue has more details from when we initially found this https://github.com/CliMA/ClimaCoupler.jl/issues/435